rclcpp/include/rclcpp/node.hpp

```diff
@@ -1430,8 +1430,11 @@ class Node : public std::enable_shared_from_this<Node>
   rclcpp::Clock::ConstSharedPtr
   get_clock() const;
 
-  /// Returns current time from the time source specified by clock_type.
+  /// Returns current time from the node time source.
```

Comment:
i agree to remove `clock_type` from here, but technically it is not always come from time source.

```suggestion
  /// Returns current time from the node clock.
```